components/base/agilex/limo_base.go

```diff
@@ -430,6 +430,7 @@ func (lb *limoBase) Properties(ctx context.Context, extra map[string]interface{}
 	return base.Properties{
 		TurningRadiusMeters: lbTurnRadiusM,
 		WidthMeters:         float64(lb.width) * 0.001, // convert from mm to meters
+		// MJ: need to add wheel circumference
```

Comment:
thoughts on excluding the agilex base from this? because right now we don't have direct access to the motors anyways correct? and we need them for this config anyways

Reply:
Return math.Nan or zero and check for that in the constructor of the movement sensor model to exclude it. Otherwise, and add a comment.